src/common/wallet-state.ts

```diff
@@ -658,7 +650,7 @@ function useWalletState(initialState?: Partial<WalletStateParams>): WalletData {
   useRecurringTimeout(async () => {
     await refreshSyncStatus()
     rendererLog.debug(`sync status`, option.getOrElseW(() => null)(syncStatusOption))
-  }, 3000)
+  }, 500)
```

Comment:
Would it make sense to extract this and similar "config" constants into a separate file?
Online check interval is another good candidate.

Reply:
@qwhex For now, I put those into constants in the same file, which is the first step. Do you think we want to have those even user-configurable (by user config)? Or just separate them to some `constants.ts` file?